packages/salesforcedx-vscode-core/src/commands/commands.ts

```diff
@@ -207,6 +207,7 @@ export abstract class SfdxCommandletExecutor<T>
     }).execute(cancellationToken);
 
     channelService.streamCommandOutput(execution);
+    channelService.showChannelOutput();
```

Comment:
This change will make all the commands open the channel output. I think it would be nice to show this on top of the notification that shows at the top for all commands but I can change this to only affect these commands for now.

Reply:
@vazexqi and I also discussed letting the CommandExecutor take a parameter for a notifier. Some commands will have different actions to show that they've executed (i.e. apex:class:create opens the file and org:open opens a browser) so it would be good to let the commands choose how they want to do that without overriding the execute method and repeating a lot of that method's code for a slight change. This will be done in a later PR.